wordclock_interfaces/templates/app.html

```diff
@@ -82,6 +82,128 @@
+                                          <v-btn
+                                            text
+                                            color="primary"
+                                            @click="menu = false"
+                                            >
+                                            Cancel
+                                          </v-btn>
+                                          <v-btn
+                                            text
+                                            color="primary"
+                                            @click="$refs.menu.save(date)"
+                                            >
+                                            OK
+                                          </v-btn>
+                                        </v-date-picker>
+                                      </v-dialog>
+                                    </v-flex>
+                                    <v-flex xs4>
+                                    </v-flex>
+                                    <v-flex xs4>
+                                      <v-dialog
+                                        ref="menutime"
+                                        v-model="menu_t"
+                                        :nudge-right="40"
+                                        :return-value.sync="time"
+                                        transition="scale-transition"
+                                        offset-y
+                                        max-width="290px"
+                                        min-width="290px"
+                                        >
+                                        <template v-slot:activator="{ on, attrs }">
+                                          <v-text-field
+                                            v-model="time"
+                                            label="Start time"
+                                            readonly
+                                            v-bind="attrs"
+                                            v-on="on"
+                                            ></v-text-field>
+                                        </template>
+                                        <v-time-picker
+                                          v-if="menu_t"
+                                          v-model="time"
+                                          full-width
+                                          v-on:change="updateScrolltext"
+                                          @click:minute="$refs.menutime.save(time)"
+                                          ></v-time-picker>
+                                      </v-dialog>
+                                    </v-flex>
+                                  </v-layout>
+                                  <v-layout row px-0>
+                                    <v-flex xs4>
+                                      <v-text-field
+                                        label="Repeat time"
```

Comment:
This sets the value for https://github.com/bk1285/rpi_wordclock/pull/210/files#diff-4d40804f6aae1831d280c31f6aa388ecd2c2372b8dccf97c2bc2defd0f7a1340R61 or does it set a "delay in seconds"?

Reply:
Maybe delay is better? This time "delay" is added to the starting time.